tests/run_translate_tests

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
The main issue I have with this file is that it doesn't actually check anything output the output, nor is integrated with the current Makefile at the root of the project.

Reply:
I've integrated it with the Makefile.  This file is just running the translation program and checking the exit code (via the `set -e`).  So just making sure the program successfully runs.